### ISSUE TITLE
Handle Alma availability API disruptions more gracefully

### ISSUE THREAD

**Issue by eporter23** [emory-libraries/blacklight-catalog#981]
Story: _As a Blacklight catalog searcher, I want to be alerted but not disrupted by data connection issues, so that I can continue using the Blacklight application for search and discovery even if the catalog data is not responsive_

We periodically receive alerts from stakeholders that the Blacklight application is suddenly returning 500 errors and certain searches or results will not load. 

We think this is caused by the Alma real time availability API not responding. So, instead of waiting for the API before we show search results or a show page, we should load the API asynchronously. While we are waiting on the API, the UI should show a loading icon like in the mockup below. If the API loads successfully, the loading message should be replaced with the current availability message and styling. If the API fails to load or returns an error, the loading message should be changed to say "Unknown".

User assistance to deal with "Unknown" availability will come in another ticket.

**Mockups:**
[Wireframe showing a new "loading" indicator for Access & Availability](https://docs.google.com/drawings/d/1Q_QjivBxuTa4xjLPv0_u-rc2HTaOEaeSjJxIgzGUqk8/edit)
[Codepen showing loader styles](https://codepen.io/eporter23/pen/vYZOJeq) : needs a proportionately [small spinner style treatment such as from Bootstrap](https://getbootstrap.com/docs/4.4/components/spinners/#size)

This alternate state indicator should be shown instead of the badge + button combination we usually present on the search results and the single item view pages if we are experiencing issues with retrieving Alma availability.

**Comment by lovinscari** [comment]
Looks good.
